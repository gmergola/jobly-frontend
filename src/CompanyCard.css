.card{
  width: 400px;
  margin: 25px;
  text-align: center;
}