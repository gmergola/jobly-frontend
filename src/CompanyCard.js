import React from 'react';
import { Link } from "react-router-dom";
import "./CompanyCard.css"
/**CompanyCard: Child component to CompanyList
 * renders information about company
 */

function CompanyCard({ name, description, handle }) {
  return (
    <div className="card">
      <div className="card-body">
        <h1 className="card-title">{name}</h1>
        <p className="card-text">{description}</p>
      </div>
      <Link className="card-link" to={`/companies/${handle}`}>See details for <b>{name}</b></Link>
    </div>
  );
}

export default CompanyCard;