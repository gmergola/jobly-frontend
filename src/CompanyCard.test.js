import React from "react";
import { render } from "@testing-library/react";
import CompanyCard from "./CompanyCard";
import { MemoryRouter } from "react-router";


it("renders without crashing", function() {
  render(
    <MemoryRouter>
        <CompanyCard />
    </MemoryRouter>
  );
});

it("matches snapshot", function() {
  const { asFragment } = render(
    <MemoryRouter>
      <CompanyCard />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
