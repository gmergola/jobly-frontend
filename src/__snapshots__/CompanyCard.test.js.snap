// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-body"
    >
      <h1
        class="card-title"
      />
      <p
        class="card-text"
      />
    </div>
    <a
      class="card-link"
      href="/companies/undefined"
    >
      See details for 
      <b />
    </a>
  </div>
</DocumentFragment>
`;
